content/en/opentelemetry/guide/otel_api_tracing_interoperability.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Interoperability of OpenTelemetry API and Datadog instrumented traces
+kind: guide
+further_reading:
+- link: "/tracing/trace_collection/otel_instrumentation/"
+  tag: "Documentation"
+  text: "Custom Instrumentation with OpenTelemetry API"
+- link: "/tracing/trace_collection/trace_context_propagation/"
+  tag: "Documentation"
+  text: "Trace Context Propagation in Datadog"
+
+---
+
+## Overview
+
+Datadog tracing libraries provide an implementation of the [OpenTelemetry API][1] for instrumenting your code. This means you can maintain vendor-neutral instrumentation of all your services, while still taking advantage of Datadog's native implementation, features, and products. 
+
+By [instrumenting your code with OpenTelemetry API][2]:
+
+- Your code remains free of vendor-specific API calls.
+- Your code does not depend on Datadog tracing libraries at compile time (only runtime).
+- Your code does not use the deprecated OpenTracing API.
+- The traces produced by your running code can be processed, analyzed, and monitored alongside Datadog traces and in Datadog proprietary products such as [Continuous Profiler][3], [Data Streams Monitoring][4], [Application Security Management][5], and [Live Processes][6].
+
+## W3C trace context propagation
+
+To facilitate seamless handling of OpenTelemetry trace data within Datadog, and correlating it with trace data generated by Datadog instrumentation, the most recent versions of Datadog tracing libraries support both [Datadog (`datadog`) and W3C (`tracecontext`) propagation styles][8] by default. [Update your runtime tracing library dependencies][7] to the latest version.
+
+Using this context propagation style allows Datadog tracers to operate in the same application environment with OpenTelemetry SDKs and other W3C compliant tracers.
+
+
+## 128-bit trace IDs
```

Comment:
In this section can we specifically say "trace IDs" everywhere we refer to an "ID"? In OpenTelemetry, trace ID's are 128-bit and span ID's are 64-bit, so I want to avoid any confusion